packages/bridge-controller/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bump `@metamask/controller-utils` to `^11.9.0` ([#5812](https://github.com/MetaMask/core/pull/5812))
 
+### Added
```

Comment:
Should be before the Changed section above to fix the lint step failing in the pipeline